openlibrary/templates/type/edition/view.html

```diff
@@ -497,3 +502,57 @@ <h2 class="home-h2"><a href="$page.url()/lists">$_('Lists containing this Book')
       </tbody>
     </table>
 </div>
+
+$def get_identifier(label):
+    $ identifier = list(values[0].value for name, values in edition.get_identifiers().multi_items() if values[0].label == label)
+    $ identifier = identifier[0] if identifier else None
+    $return identifier
+
+$def get_url(label):
+    $ url = list(values[0].url for name, values in edition.get_identifiers().multi_items() if values[0].label == label)
+    $ url = url[0] if url else None
+    $return url
+    
+
+<script type="application/ld+json">
+{
+  "@context": "http://schema.org/",
+  "@type": "Book",
+  "@id": "$base_url/$work.key",
+  "url": "$base_url/$work.key/$book_title",
```

Comment:
Too many slashes here (work.key already has a slash)

```suggestion
  "@id": "$base_url$work.key",
  "url": "$base_url$work.key/$book_title",
```